speech/README.md

```diff
@@ -22,9 +22,10 @@ Usage:
     Usage: ruby speech_samples.rb <command> [arguments]
 
     Commands:
-      recognize               <audio-file.raw>
-      async_recognize         <audio-file.raw>
-      async_recognize_results <operation name>
+      recognize           audio-file.raw
```

Comment:
I was gonna look at [Language](https://github.com/GoogleCloudPlatform/ruby-docs-samples/tree/master/language) as an example of another that has GCS options, but... whoops!  README needs to be updated.

I don't want to remove "gs://bucket/something" because I don't think that's an obvious format.

Not sure what to put for these, because all 4 commands do the same thing...

```
Usage: ruby speech_samples.rb <command> [arguments]

Commands:
    recognize           audio-file.raw                Transcribe file
    recognize_gcs       gs://bucket/audio-file.raw    Transcribe file
    async_recognize     audio-file.raw                Transcribe file
    async_recognize_gcs gs://bucket/audio-file.raw    Transcribe file
```

Would you like an update to that ^ ?